lnrpc/rpc.proto

```diff
@@ -1703,6 +1703,15 @@ message PendingUpdate {
     uint32 output_index = 2;
 }
 
+message ReadyForPsbtFunding {
+    string funding_address = 1;
```

Comment:
How about making this a one-of that also includes a raw PSBT? Just to support all the flows we have in mind.

Reply:
I changed the response to always contain the address, amount and PSBT so the user has the choice of what to use. A `oneof` doesn't really make sense IMO as it's the same information, just encoded differently twice.